src/webextension/widgets/input.css

```diff
@@ -78,17 +79,25 @@
   color: #003eaa;
 }
 
+.field-text,
+.label-text {
+  color: #0c0c0d;
+  text-align: left;
+  line-height: 1.5;
+  letter-spacing: 0.2px;
+}
+
 .field-text {
   /* This lines up our labels with the text in <input> elements. */
-  margin-top: 7px;
   min-height: 25px;
-  font-size: 13px;
+  font-size: 15px;
+  font-weight: bold;
```

Comment:
This is a design thing, but I'm not a fan of these changes. When implementing this originally, I was very careful to make sure that the height of the `<FieldText/>` elements were the same as the height of the `<Input/>` elements. This is to ensure that when you click "Edit" on an entry you're viewing, the fields don't change position as you go from view-mode to edit-mode (unless you have fields long enough to wrap, which should be rare). By making these a different size, we lose that.

Reply:
>This is to ensure that when you click "Edit" on an entry you're viewing, the fields don't change position as you go from view-mode to edit-mode

@changecourse what do you think about this consistent size so there's less jarring "jumps" in sizes between view/edit modes?

Reply:
As shown here: https://mozilla.invisionapp.com/share/W3EU997YN#/262163359_1-1-2_Lockbox_Home_-_Edit_Entry

I have both the read-only text and the form field text at 15, and am trying to keep the fields from changing position to drastically, no? Toggle between the two screens for edit vs. view for context. Am I missing something?

Reply:
I looked into it more and the Photon spec does say that inputs should have 15px text (ours is 13px), *but* actual Photon UI that I can find in Firefox uses 13px* (e.g. about:addons, about:preferences). Since one of the goals is to have this look like it "belongs" in Firefox, I lean towards following Firefox's lead for the time being. We should also bring this up with the Photon designers, since I'm not really clear on why there's a difference here.

* Technically 13.3167px.